src/psl.c

```diff
@@ -56,6 +56,10 @@
 # include <unistd.h>
 #endif
 
+#if defined(_MSC_VER) && ! defined(ssize_t)
+typedef SSIZE_T ssize_t;
```

Comment:
What about basetsd.h here ? It might be included by winsock2.h, but we shouldn't rely on it...

Reply:
Alright, added the include here as well.